README.md

```diff
@@ -106,25 +106,30 @@ As it turns out, people usually find this document before they sign a contract w
-| RT / non-project meeting                   | phone calls and Slack conversations that are related to your work but not directly project-related |
-| RT / virtual assistant                     | things related to organizing the equipment needed in the company (like laptops or any necessary office stuff or hardware) |
+| RT / django cookiecutter template          | development of our internal django template |
+| RTO&nbsp;/&nbsp;internal&nbsp;infrastructure&nbsp;management    | anything related to getting your workspace ready; creating encrypted partitions, installing vms or any software needed to work and **not connected to any specific project**. If it's related to any internal or external project then please bill the time on that project |
+| RT / non-project meeting                   | phone calls and Slack conversations that are related to your work but not directly project-related. These are very rare, almost non-existent. |
+| RTO / virtual assistant                    | things related to organizing the equipment needed in the company (like laptops or any necessary office stuff or hardware) or other VA tasks |
 | RT / other internal development            | spending time on improving internal infrastructure like onboarding, training or any other company related stuff that might be helpful for others |
-| RT / pre-sales                             | any pre-work needed to be done **before** the contract is signed with the customer |
-| RT / recruitment                           | time spent on actively recruiting other people |
-| RT / sociocracy                            | for s3 meetings, but also for chatting on the channel and for tuners |
-| RT / special assignment from CEO           | bill like this only when specifically asked to |
-| RT / onboarding                            | only during onboarding phase (going through [training](training.md) materials) |
+| RT / security training                     | when you join at some point you'll get a security training that takes an hour or so - we have all staff complete it (and staff of our clients too) to harden ourselves against fraud, hackers etc |
+| RTO / sales                                | any pre-work needed to be done **before** the contract is signed with a client |
+| RTO / nurturing client relationship        | this is like sales, but for (brief) discussing an upcoming project of a client with an existing contract OR theoretically after-sales support on a fixed price project (we haven't had any since like 2019 and the guarantee expired already on these) |
+| RTO / recruitment                          | time spent on actively recruiting other people |
+| RT / recruitment                           | time spent on improving the recruitment process |
+| RTO / sociocracy meeting                   | for s3 meetings, but also for chatting on the channel |
+| RT / sociocracy development                | for s3 tuners |
```

Comment:
Explain "s3 tuners"

Reply:
again, this is not the place to explain it. The purpose of this section is different.